src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -205,22 +205,33 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
         
         ______TS("feedback session submission email");
 
-        Calendar time = Calendar.getInstance();
+        Calendar time = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
         time.set(Calendar.DATE, 4);
         time.set(Calendar.MONTH, 8);
         time.set(Calendar.HOUR_OF_DAY, 5);
         time.set(Calendar.MINUTE, 30);
         EmailWrapper email = new EmailGenerator().generateFeedbackSubmissionConfirmationEmailForStudent(
-                session, student1, TimeHelper.formatTime12H(time.getTime()));
+                session, student1, time);
         subject = String.format(EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject(), course.getName(),
-                                session.getFeedbackSessionName());
-        verifyEmail(email, student1.email, subject, "/sessionSubmissionConfirmationEmailForStudent.html");
+                session.getFeedbackSessionName());
+        verifyEmail(email, student1.email, subject,
+                "/sessionSubmissionConfirmationEmailPositiveTimeZone.html");
 
+        session.setTimeZone(-9.5);
         email = new EmailGenerator().generateFeedbackSubmissionConfirmationEmailForInstructor(session,
-                instructor1, TimeHelper.formatTime12H(time.getTime()));
+                instructor1, time);
         subject = String.format(EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject(), course.getName(),
-                                session.getFeedbackSessionName());
-        verifyEmail(email, instructor1.email, subject, "/sessionSubmissionConfirmationEmailForInstructor.html");
+                session.getFeedbackSessionName());
+        verifyEmail(email, instructor1.email, subject,
+                "/sessionSubmissionConfirmationEmailNegativeTimeZone.html");
+
+        session.setTimeZone(0.0);
+        email = new EmailGenerator().generateFeedbackSubmissionConfirmationEmailForInstructor(session,
+                instructor1, time);
+        subject = String.format(EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject(), course.getName(),
+                session.getFeedbackSessionName());
+        verifyEmail(email, instructor1.email, subject,
+                "/sessionSubmissionConfirmationEmailZeroTimeZone.html");
```

Comment:
This is aesthetic and you can ignore if you want, but you may consider:
- Aligning `[s]ession` with `[E]mailType`
- Moving this line to the line before since it does not exceed the line length limit

Applies also to the other instances of this routine. Essentially, look at the changes introduced in lines 216-218 and try to do it such that the change is minimized.